src/Cli/Microsoft.DotNet.Cli.Utils/NuGetPackageInstaller.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using NuGet.Common;
+using NuGet.Protocol;
+using NuGet.Protocol.Core.Types;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    internal class NuGetPackageInstaller
```

Comment:
Don't put this in util, this is an adapter. (although in my opinion, most of things inside util should not be in it).